JamfSync/Model/Jcds2Dp.swift

```diff
@@ -200,33 +207,453 @@ class Jcds2Dp: DistributionPoint {
         } else {
             fileUrl = file.fileUrl
         }
+        
         guard let fileUrl else { throw DistributionPointError.badFileUrl }
+        
+        
+        _ = disableSleep(reason: "start upload")
```

Comment:
Instead of ignoring the return value, check it and log a message in case it fails. We can continue anyway, but that way we have an idea of what's happening if it does fail and therefore causes an upload to fail later on.